randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1971,17 +1971,23 @@ Extra - room_id: [42]
   > Event - Serris
       All of the following:
           Any of the following:
-              # Serris Weapon Requirements
-              Charge Beam or Missiles ≥ 10
-              Missiles ≥ 5 and 20 DMG Missiles
+              # Serris Weapon Requirements (50HP)
+              Charge Beam or Missiles ≥ 20
+              Missiles ≥ 10 and 20 DMG Missiles
+              All of the following:
+                  Combat (Advanced)
+                  Any of the following:
+                      Missiles ≥ 10
+                      Missiles ≥ 5 and 10 DMG Missiles
           # Core-X Requirements
           Missiles
           Any of the following:
               # Health Requirements
-              Combat (Expert) or Normal Damage ≥ 400
-              Combat (Beginner) and Normal Damage ≥ 300
+              Normal Damage ≥ 300
+              Combat (Beginner) and Normal Damage ≥ 200
               Combat (Intermediate) and Normal Damage ≥ 150
```

Comment:
since serris appears before box, and you have after serris more chances to find more etanks in vanilla, thoughts on bumping these down to 250, 180, 110 instead?
(after multiplier these would be 375, 270, 165)